src/core/qfieldcloudprojectsmodel.cpp

```diff
@@ -1730,16 +1804,16 @@ void QFieldCloudProjectsModel::projectListReceived()
   }
 }
 
-NetworkReply *QFieldCloudProjectsModel::downloadFile( const QString &projectId, const QString &fileName )
+NetworkReply *QFieldCloudProjectsModel::downloadFile( const QString &projectId, const QString &fileName, bool fromPackage )
```

Comment:
```suggestion
NetworkReply *QFieldCloudProjectsModel::downloadFile( const QString &projectId, const QString &fileName, const QString &packageId )
```

Note one can download from multiple packages. This soon will be the case when we support multiple password credentials and therefore packages for endusers. So values could be:

- `fromPackage.isEmpty()` - download from standard files
- `! QUuid.fromString( fromPackage ).isNull()` - download from given package
- `fromPackage == QStringLiteral( "latest" )` - download from latest
- else - panic!

If you want to keep the boolean logic for now, rename it to `bool fromLatestPackage`.